From 831ebd0d8a961b3539e2731f2bcc21fb793466b3 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Tue, 24 Apr 2018 12:07:48 +0300
Subject: [PATCH 0603/1051] fix: arm64: dts: Disable AP i2c on A8040 DB

The AP i2c has no connections on A8040 DB, so scanning this bus
using i2c tools cause bus error.
Disable the AP i2c on A8040 DB so the CP i2c becomes i2c port 0.

Change-Id: I074a47df1e2324628180ed180b0311d5492861de
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-db.dts | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db.dts b/arch/arm64/boot/dts/marvell/armada-8040-db.dts
index 6640d3fa4c97..7b766125682a 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db.dts
@@ -81,11 +81,6 @@
 	};
 };
 
-&i2c0 {
-	status = "okay";
-	clock-frequency = <100000>;
-};
-
 &spi0 {
 	status = "okay";
 
-- 
2.17.1

